kompose.spec

```diff
@@ -43,13 +35,11 @@
 %global commit          822768446621d71ffd02eafd138db0d8ab4a7d0e
 %global shortcommit     %(c=%{commit}; echo ${c:0:7})
 
-Name:           golang-%{provider}-%{project}-%{repo}
-Version:        0
+Name:           kompose
+Version:        0.1.0
 Release:        0.1.git%{shortcommit}%{?dist}
-Summary:        !!!!FILL!!!!
-# Detected licences
-# - Unknown at 'LICENSE'
-License:        !!!!FILL!!!!
+Summary:        Tool to move from `docker-compose` to Kubernetes
+License:        Apache 2.0
 URL:            https://%{provider_prefix}
 Source0:        https://%{provider_prefix}/archive/%{commit}/%{repo}-%{shortcommit}.tar.gz
```

Comment:
it is easier to update to tarball between releases (sometimes it makes sense). Otherwise not.